### PR TITLE
release: v3.0.1rc1

### PR DESCRIPTION
**Pull request by lnielsen**
@inveniosoftware/architects Please have a look.

### CODE REVIEW

File: docs/releases/v3.0.1.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- **Invenio-App:** Fixed compatibility issue with latest release of
+  Flask-Talisman.
+
+- **Invenio-DB:** Fixed a compatibility issue with newly released
+  SQLAlchemy-Continiuum.
+
+- **Invenio-Search:** Fixed a compatibility issue with
+  elasticsearch-dsl>=6.2.0.
+
+- **Pytest-Invenio:** Fixed a problem which caused Invenio modules to be loaded
```
Comment (slint) [inline docs/releases/v3.0.1.rst]: Maybe grouping these might look a bit better and be easier to read? (e.g. like Django does on a per-"module" level: https://docs.djangoproject.com/en/2.1/releases/2.1/#minor-features)

Comment (lnielsen) [reply docs/releases/v3.0.1.rst]: Actually they keep it flat for patch releases: https://docs.djangoproject.com/en/2.1/releases/2.1.1/

My thinking was just that in most cases there's one or two bullets per module.

File: setup.py [hunk 2]
```diff
@@ -40,18 +40,18 @@
...
         'invenio-userprofiles>=1.0.1,<1.1.0',
     ],
     'metadata': [
-        'invenio-indexer>=1.0.0,<1.1.0',
+        'invenio-indexer>=1.0.1,<1.1.0',
         'invenio-jsonschemas>=1.0.0,<1.1.0',
         'invenio-oaiserver>=1.0.0,<1.1.0',
         'invenio-pidstore>=1.0.0,<1.1.0',
-        'invenio-records-rest>=1.1.0,<1.3.0',
+        'invenio-records-rest>=1.1.1,<1.2.0',
```
Comment (lnielsen) [inline setup.py]: Note, here I'm staying with ``<1.2.0`` and not ``<v1.3.0`` which was merged in master

Comment (egabancho) [reply setup.py]: I think `1.1.1` was never released, travis was failing, what about `1.1.2` as minimum?

File: docs/releases/v3.0.1.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  despite the related test fixtures not being used, thus causing import errors
+  if the module was not installed.
+
+- **Pytest-Invenio:** Fixed a compatibility issue with pytest v3.8.x.
```
Comment (fenekku) [inline docs/releases/v3.0.1.rst]: To be more precise, it was starting with pytest 3.8.1 (https://github.com/pytest-dev/pytest/issues/3941) (3.8.0 was fine) :nerd_face: 

```suggestion
- **Pytest-Invenio:** Fixed a compatibility issue with pytest v3.8.1+.
```

**Pull request merged**